mysite/missions/templates/missions/pipvirtualenv/installing_packages.html

```diff
@@ -0,0 +1,110 @@
+{% extends 'missions/pipvirtualenv/base.html' %}
+{% comment %}
+# This file is part of OpenHatch.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+
+{% block mission_main %}
+<div class='submodule fat'>
+  <div class='head'>
+    <h3>Installing packages and creating virtualenvs</h3>
+  </div>
+  <div class="body">
+
+  <h3>Creating virtualenvs</h3>
+  <p>At the terminal, you can create a new virtual environment by typing
+  <kbd>mkvirtualenv</kbd>, followed by a name of your choice for the new
+  virtualenv.</p>
+
+  <p>Now, open up the terminal, type the following command and press Enter:</p>
+  <kbd>mkvirtualenv TestEnv</kbd>
+
+  <p>The above command will create a new virtual environment called <kbd>
+  TestEnv</kbd>. The name of the virtual environment can be anything you like.
+  The output of the above command should look something like this:</p>
+  <pre>
+    New python executable in TestEnv/bin/python
+    Installing setuptools, pip...done.
+  </pre>
+
+  <p>After running the <kbd>mkvirtualenv</kbd> command above, the name of the
+  virtualenv you created(<kbd>TestEnv</kbd>, in this case) will appear
```

Comment:
I think it would be good to remove the parens on this line. I think the parens should only be around (TestEnv) in this paragraph when used to illustrate that the parens tell the user that the virtual environment has been activated.